### PR TITLE
Add 'useDisconnect' hook

### PR DESCRIPTION
**Pull request by bisgardo**

### CODE REVIEW

File: packages/react-components/src/useWalletConnectorSelector.ts
Comment (bisgardo) [inline packages/react-components/src/useWalletConnectorSelector.ts:1]: Should also have a field for indicating if the connector is initializing?

Comment (bisgardo) [reply packages/react-components/src/useWalletConnectorSelector.ts:1]: And if initialization failed...

File: packages/react-components/src/useWalletConnectorActivation.ts
Comment (bisgardo) [inline packages/react-components/src/useWalletConnectorActivation.ts:1]: Should be named `WalletConnectorTypeActivation`? As it's really more closely reflecting the activation of a connector *type*.

Should also have a field for indicating whether the connector is initializing? ... And whether initialization failed?

**Comment by soerenbf** [comment]
Closing this as it's outdated

**Pull request closed without merging**